packages/react-core/src/components/Tooltip/Tooltip.tsx

```diff
@@ -230,20 +230,27 @@ export const Tooltip: React.FunctionComponent<TooltipProps> = ({
       hide();
     }
   }, [isVisible]);
+  React.useEffect(() => {
+    clearTimeouts([transitionTimerRef, hideTimerRef]);
+    hideTimerRef.current = setTimeout(() => {
+      setOpacity(0);
+      transitionTimerRef.current = setTimeout(() => setVisible(false), animationDuration);
+    }, exitDelay);
+  }, [exitDelay]);
   const show = () => {
     clearTimeouts([transitionTimerRef, hideTimerRef]);
     showTimerRef.current = setTimeout(() => {
       setVisible(true);
       setOpacity(1);
     }, entryDelay);
   };
-  const hide = () => {
-    clearTimeouts([showTimerRef]);
+  const hide = React.useCallback(() => {
```

Comment:
`useCallback` is a way to memoize the execution of a function unless the value of one of its dependencies changes. 

In this case, I want to be sure that the _current_ hideTimerRef is always using the current `exitDelay`, and since that can be updated by the user dynamically (as in the case of the copy buttons) i need to basically redefine the body of this `hide` callback function whenever the `exitDelay` is updated.

Reply:
Gotcha, thanks for that info!